tests/scan/test_scannable.py

```diff
@@ -179,3 +183,53 @@ def test_patch_max_size():
     files = list(c.get_files())
 
     assert len(files) == 0
+
+
+@pytest.mark.parametrize("absolute_path", (True, False))
```

Comment:
Instead of handling absolute paths, I think it would be simpler and safer to always store paths relative to the base scan dir. Given this file hierarchy:

```
/foo
- /bar
  - /baz
    - file1.txt
  - file2.py
```

Then if a command is run on /foo/bar, the archive contains:

- baz/file1.txt
- file2.py

Reply:
Yes, we discussed that with Henri. I will make a follow-up issue to remove the unnecessary parent directories at some point before sending the archive (either when building the archive or even before)

Reply:
I changed the signature to use a root_path for the base scan dir and expect the files to have a relative path to this root_path. Also moved it out of the Files class since it now needs a tuple of data Path/Files which need to be consistent